components/Tabs/Tabs.js

```diff
@@ -66,7 +71,14 @@ class TabCard {
 
 - With your selection in place, now chain a .forEach() method onto the tabs variable to iterate over the DOM NodeList
 
-- In your .forEach() method's callback function, return a new instance of TabLink and pass in each tab as a parameter
+- In your .forEach() method"s callback function, return a new instance of TabLink and pass in each tab as a parameter
 
 */
-let tabs = document.querySelectorAll();
\ No newline at end of file
+// let tabs = document.querySelectorAll(".tab");
+// tabs.forEach(tab => new TabLink(tab));
+
+
+let tabs = document.querySelectorAll(".tab");
+tabs = Array.from(tabs).map( (tab) => new TabLink(tab))
+
```

Comment:
The above method chaining displays sophisticated understanding of how these methods operate.